utils/utils_darwin.go

```diff
@@ -0,0 +1,9 @@
+package utils
+
+import (
+	"golang.org/x/sys/unix"
+)
+
+func syncfs(fd uintptr) error {
+	return unix.Fsync(int(fd))
```

Comment:
This isn't the same as Syncfs, this only syncs one file, not everything on the filesystem.